objects/UniversalBenefit.js

```diff
@@ -21,6 +24,32 @@ export const StatusCodes = {
   Pending: 'Pending',
 }
 
+export const TypeCodes = {
+  CPPBeneficial: 'CPPBeneficial',
+  CPPRetirement: 'CPPRetirement',
+  CPPDisability: 'CPPDisability',
+  EIUnknown: 'EIUnknown',
+}
+
+export function CreateBenefitSummary(type, value, status) {
+  let benefitSummary = {
+    type: type, // will define what header text to go with it and any links if applicable
+    value: value, // a date or amount, defined by the type
+  }
+  //only add the status object if it exists
+  if (status != undefined) {
```

Comment:
Same as null a null check here. I think you can change this as `if (!status) {`. If not, use triple equals :)